config/env.sample

```diff
@@ -0,0 +1,5 @@
+PORT=5005
+CERT_FILE="config/public-cert.pem"
+KEY_FILE="config/private-key.pem"
```

Comment:
Ok. This would be a good chance to drop them from proxy.js then, but either way is fine.